ccd-definition/CaseTypeTab/CareSupervision/9_Documents.json

```diff
@@ -50,7 +50,8 @@
     "TabLabel": "Documents",
     "TabDisplayOrder": 9,
     "CaseFieldID": "hearingFurtherEvidenceDocuments",
-    "TabFieldDisplayOrder": 5
+    "TabFieldDisplayOrder": 5,
+    "FieldShowCondition": "hearingFurtherEvidenceDocuments!=\"\""
```

Comment:
Isn't this condition the default implementation of CCD?

Reply:
Actually I remember, if the list you send is empty then it can show. Maybe it would be best to send  null instead of an empty list 🤔

Reply:
discussed and removed